rumqttc/examples/tls.rs

```diff
@@ -15,12 +15,11 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     // Use rustls-native-certs to load root certificates from the operating system.
     let mut root_cert_store = tokio_rustls::rustls::RootCertStore::empty();
-    for cert in rustls_native_certs::load_native_certs().expect("could not load platform certs") {
-        root_cert_store.add(&tokio_rustls::rustls::Certificate(cert.0))?;
-    }
+    root_cert_store.add_parsable_certificates(
+        rustls_native_certs::load_native_certs().expect("could not load platform certs"),
+    );
```

Comment:
using `add(_)` to add individual cert give us better control, as `add_parsable_certificates` might ignore some certs. 

as this is just an _example_, it doesn't matter tbh, so feel free to mark this as resolved if you wish :)

Reply:
I think ignoring invalid certificates is the right thing to do, rather than crashing just because someone happens to have some root certificates in their system certificate store which we can't parse. They probably don't have control over the certificates provided by the OS anyway, and skipping some doesn't make things any less secure.

Reply:
we were simply calling `.unwrap()` [here](https://github.com/bytebeamio/rumqtt/pull/790/files#diff-a7c14b22199b25375255ee5a3d137a7f77ad9fdde94ff211cd27c6441f3cbd1dR369) in `lib.rs`, hence I asked.

Reply:
Good point, that should probably change too.